src/components/Navigation/JoinCreateLeagueBtn/JoinCreateLeague.jsx

```diff
@@ -0,0 +1,222 @@
+import React, { Component } from 'react';
+import './JoinCreateLeague.css';
+import { PlusSquare } from 'react-feather'
+import Modal from 'react-responsive-modal';
+import Select from 'react-select';
+import Button from '../../Button/Button';
+import DayPicker from 'react-day-picker';
+import 'react-day-picker/lib/style.css';
+import { withAlert } from 'react-alert'
+import { createLeague, joinLeague } from '../../../api/api';
+
+const selectOptions = [
+  { value: 'join', label: 'Join a League' },
+  { value: 'create', label: 'Create a League' }
+]
+
+class JoinCreateLeague extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isModalOpen: false,
+      selectedOption: selectOptions[0],
+      inviteCode: '',
+      name: '',
+      leagueName: '',
+      buyingPower: 0,
+      selectedStartDay: null,
+      selectedEndDay: null,
+      createLeagueErrorMsg: ''
+    };
+  }
+
+  openModal = () => {
+    this.setState({ isModalOpen: true });
+  };
+
+  closeModal = () => {
+    this.setState({ isModalOpen: false });
+  };
+
+  handleSelectChange = (selectedOption) => {
```

Comment:
Might be nice to make this more specific (what's being selected?)

Reply:
It's super long now!